g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh

```diff
@@ -0,0 +1,77 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: go-hclog
+# Version	: ff2cf002a8dd
```

Comment:
is this for the top-of-tree or for an older version? If its top-of-tree, can we remove the commit # from the script name?

Reply:
The version was mentioned in the list is as "**v0.0.0-20180709165350-ff2cf002a8dd**". It's commit id is ff2cf002a8dd. 
Could you please advise if I need to use v.0.0.0 in the script name?

Reply:
can we test it for the default branch and if it works, remove the version altogether from the script name?

Reply:
Removed second script. First script is working with main(master) branch also.